<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestProjectClass</class>
 <widget class="QMainWindow" name="TestProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>421</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TestProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>http://www.random.org/integers/?num=1&amp;min=1&amp;max=6&amp;col=1&amp;base=10&amp;format=plain&amp;rnd=new</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Phonon::VolumeSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Phonon::VideoPlayer" name="videoPlayer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh BD</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Music path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="MusicPathTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="TrueRndCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use True random (internet connection required)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Phonon::SeekSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ButtonPause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PreviousButton">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlayButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="testproject.qrc"/>
 </resources>
 <connections/>
</ui>
